src/pybind/mgr/deepsea/module.py

```diff
@@ -121,12 +121,16 @@ def available(self):
 
 
     def get_inventory(self, node_filter=None):
+        # Note that this will raise an exception (e.g. if the salt-api is down,
```

Comment:
Would it make sense to add this as a docstring instead?

Reply:
Done